### PR TITLE
feature/kaju

### PR DESCRIPTION
**Pull request by kkaajjaall**
practice going on.....

### CODE REVIEW

File: second.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1> ITHQ -India </h1>
```
Comment (ithqindia) [inline second.html]: Should be  `<h1>ITHQ - India</h1>`

File: second.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ithqindia) [inline second.html]: Change Document to ITHQ - India

**Comment by kkaajjaall** [comment]
second try done

**Comment by kkaajjaall** [comment]
second try done..

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: second.html (hunk 1 above)
Comment (kkaajjaall) [reply second.html]: code updated.

**Review by ithqindia — APPROVED** [review]

**Pull request merged**